### ISSUE TITLE
chromecast.disconnect() causes always a warning?

### ISSUE THREAD

**Issue by martinpoirot** [vitalidze/chromecast-java-api-v2#120]
its a really small issue, but is it possible to avoid the warning "Got IOException while reading due to stream being closed (stop=true), caused by java.net.SocketException: Socket closed" after Chromecast.disconnect()?

we simply try to stream a url to the chromecast by:

``` 
ChromeCast c = new ChromeCast( ipAddr);
c.connect();
try {
        Status chromeState = c.getStatus();
       ...
        c.launchApp(CHROMECAST_STANDARD_RECEIVER_APP);
      ...
	MediaStatus mediaStatus = c.load( "title", null, url, MediaType.IMAGE_JPEG_VALUE );
} finally {
	if ( c.isConnected()) {
		c.disconnect();	
	}  
}
```
-> we always get this warning

after short debuging, seems like chromecast.disconnnect() calls channel.close() ->
``` 
public void close() {
...              
  if (reader != null) {
    reader.stop = true;
  }
  if (socket != null) {
    socket.close();
  }
```

an closes the socket, after setting the reader thread stop=true flag.

the Reader Thread seems to wait in inputstream.read() and closing the socket causes an IOException, as expected, but the Exception Handling log it with a warning:

```
} catch (IOException ioex) {
  if (stop) {
    warn("Got IOException while reading due to stream being closed (stop=true)", ioex);
    continue;
  }
```

so, we guess, there is no way, to avoid this "Exception" and the warning?

-> the only solution we could imagine, is to increase the logger level up to ERROR, but is that useful? we are than completely blind to other meaningful warnings?

**Comment by vitalidze** [comment]
@martinpoirot thank you for reporting. It seems that the `IOException` is definitely expected every time you disconnect from the device. I believe the correct and safe way to handle this is to lower the logging level even more - to the `DEBUG` level. Hope that works for you.

**Comment by martinpoirot** [comment]
It would be very appreciated, to have this specific warning on debug level -> so, raising the overall log level to error for the cannel class could be avoided .... thanks a lot!

**Comment by vitalidze** [comment]
I have just released a new version of the library (`0.11.3`) with the fix.
